changelogs/unreleased/6375-add_exclude_event_to_get_resource_events.yml

```diff
@@ -0,0 +1,7 @@
+description: Add 'exclude_change' argument to 'get_resource_events' to be able to exclude some types of changes from the results.
+issue-nr: 6375
+change-type: minor
+destination-branches: [master]
```

Comment:
Why do we scope this to master only?

Reply:
@arnaudsjs I though it was a new feature for iso7. But indeed maybe it should go into  iso6 as well. Do you know? otherwise i'll ask on slack

Reply:
You can ask on Slack.